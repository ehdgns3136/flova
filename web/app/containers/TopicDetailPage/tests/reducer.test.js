//
// import { fromJS } from 'immutable';
import topicDetailPageReducer from '../reducer';

describe('topicDetailPageReducer', () => {
  it('returns the initial state', () => {
    expect(topicDetailPageReducer(undefined, {})).toMatchSnapshot();
  });
});
