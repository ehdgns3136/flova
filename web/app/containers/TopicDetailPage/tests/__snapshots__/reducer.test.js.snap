exports[`topicDetailPageReducer returns the initial state 1`] = `
Object {
  "activeItem": "popularQuestions",
  "contents": Array [],
  "isContentsLoading": true,
  "isTopicDetailLoading": true,
  "topic": null,
}
`;
